tests/tests/test_redirects.py

```diff
@@ -88,16 +88,26 @@
     ("/unfuck/", "https://www.mozilla.org/firefox/unfck/?redirect_source=firefox-com", 301),
     ("/love/", "https://www.mozilla.org/firefox/unfck/?redirect_source=firefox-com", 301),
     ("/liebe/", "https://www.mozilla.org/firefox/unfck/?redirect_source=firefox-com", 301),
```

Comment:
I noticed that in the tests for the previously set redirects we add a trailing `/`. Maybe we want to update them to remove this optional character?

Reply:
Sure. I need to update them for #19 anyway.